guides/common/modules/ref_shellhooks.adoc

```diff
@@ -0,0 +1,10 @@
+[id="shellhooks_{context}"]
+= Shellhooks
+
+Optionally, {SmartProxy} Shellhooks plugin can be installed which exposes executables using a REST HTTPS API.
+Scripts must be placed in `/var/lib/foreman-proxy/shellhooks` as executables with only alphanumeric characters and underscores in the name.
+The HTTPS payload is passed using standard input, optional command line arguments can be provided using X-Shellhook-Arg-1 _to_ N.
+HTTP method must be POST, an example URL would be: `https://{smartproxy.example.com}:{smartproxy_port}/shellhooks/my_script`.
```

Comment:
This is an incorrect attribute. 
Please reference the attributes-base.adoc file and update.